cumulusci/core/config/base_config.py

```diff
@@ -23,7 +30,47 @@ def _load_config(self):
         """Subclasses may override this method to initialize :py:attr:`~config`"""
         pass
 
+    @classmethod
+    def allowed_names(cls) -> set:
+        properties = set(dir(cls))
+        annotations = set(getattr(cls, "__annotations__", {}))
+        return properties.union(annotations)
+
+    # long term plan is to get rid of this
     def __getattr__(self, name):
```

Comment:
A docstring explaining the policy we're trying to enforce would be helpful.

Reply:
Fixed in ee235fe01